CHANGELOG.md

```diff
@@ -17,6 +17,9 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
   -------------------------------------------------------------------
 ## [Unreleased]
 
+## [1.14.1] 2021-01-29
```

Comment:
Hey, thanks for the change. In future you can just add the entry under the unreleased section, since these are unreleased changes. When i cut the trunk into a release the version number will be decided. In this case it'll actually be `1.15.0` :)